docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Getting started
+order: 0
+slug: getting-started
+lang: en
+layout: tutorial
+permalink: /future/getting-started
+---
+
+# Getting started <span class="label-beta">BETA</span>
+
+<div class="section-content">
+This guide will walk you through introducing you to the [next-generation platform](/bolt-js/future/getting-started#next-gen). It will also cover how to get started with your next-generation platform experience with Bolt JS by setting up the Slack CLI and also installing the required dependencies.
+</div>
+
+---
+### What is the next-generation platform? {#next-gen}
+
+The next-generation platform is an exciting new way to develop apps for Slack! It is focused on 2 components:
+1. **Flexible functionality**: You can now create modular, shareable, and remixable building blocks within your apps that make it easier than ever to connect and reuse functionlity. The primary components of this new functionality are [built-in](/bolt-js/future/built-in-functions) and [custom functions](/bolt-js/future/custom-functions), [workflows](/bolt-js/future/workflows), and [triggers](/bolt-js/future/triggers).
+2. **Quick setup**: With our next-generation developer tools, you can quickly build secure, scalable apps. One of the tools is the [Slack CLI](https://api.slack.com/future/tools/cli), which allows faster development by allowing you to create and scaffold apps all from the command line. Another tool are [app manifests](/bolt-js/future/app-manifest), which are configuration files for your apps that can be shared and reused to quickly set up applications.
+
+#### Limitations
+
+Currently for Bolt JS, you can only create remote functions and apps which can be hosted and deployed on [Heroku](https://www.heroku.com/). Next-generation applications not built on Bolt frameworks (such as those built on [Deno](https://deno.land/)) can support non-remote functions and apps, meaning you can deploy them to secure and managed Slack infrastructure. Additionally, Bolt apps built on the next-generation platform currently do not support [Datastores](https://api.slack.com/future/datastores) that are hosted on Slack infrastructure.
```

Comment:
Can we standardardize places that say Bolt JS and call it Bolt for Javascript instead?